spring-core/src/test/java/org/springframework/core/convert/support/GenericConversionServiceTests.java

```diff
@@ -61,11 +62,16 @@
  * @author Phillip Webb
  * @author David Haraburda
  * @author Sam Brannen
+ * @author Yanming Zhou
  */
 class GenericConversionServiceTests {
 
-	private final GenericConversionService conversionService = new GenericConversionService();
+	private GenericConversionService conversionService;
 
+	@BeforeEach
+	void init() {
+		conversionService = new GenericConversionService();
+	}
```

Comment:
This test class uses JUnit Jupiter with the default `@TestInstance` lifecycle mode which is `PER_METHOD`, meaning that the test instance will be instantiated for each test method.

Consequently, the `GenericConversionService` will be recreated for every instance/test.